admin/commands/summary.go

```diff
@@ -153,9 +153,9 @@ func addClientData(ctx context.Context, zipW *zip.Writer) {
 }
 
 // addServerData adds logs.zip from PMM Server to zip file.
-func addServerData(ctx context.Context, zipW *zip.Writer) {
+func addServerData(ctx context.Context, zipW *zip.Writer, usePprof bool) {
 	var buf bytes.Buffer
-	_, err := client.Default.Server.Logs(&server.LogsParams{Context: ctx}, &buf)
+	_, err := client.Default.Server.Logs(&server.LogsParams{Context: ctx, Pprof: &usePprof, HTTPClient: nil}, &buf)
```

Comment:
We can omit `HTTPClient: nil`

Reply:
Yes we can; i have added it because of linter warning.

Reply:
What linter? Before we disabled `exhaustivestruct` liter exact for such useless warnings.